### DISCUSSION TITLE
Is there any written communication from Gmail regarding future support for Inbox SDK?

### DISCUSSION THREAD

**Discussion by anishvmonsy** [InboxSDK/InboxSDK#1007] [Q&A] [answered]
Hi folks,
I was planning to create an extension targetting Gmail users and I came across Inbox SDK that I could possibly use. I found the following article:
https://news.ycombinator.com/item?id=33164922
In this article, there is a claim that Google is super supportive of the SDK and early access to several builds is given to ensure the SDK doesn’t break when they make updates to Gmail. 

Has there been any written communication from Google to ensure future support? If that is shared, this would give confidence to existing/future users of Inbox SDK regarding its stability long term.

**Comment by aleemstreak** [comment, marked as answer]
There isn't written communication that we would feel comfortable publishing. Though we have a plan to do a guest blog post on the chrome extension blog. 

I think the major thing to think about is that Streak is an 8 figure revenue business that is wholly dependent on the InboxSDK so there is a lot of incentive for us to keep the SDK well maintained.
